frontend/src/router/layouts/MainLayout.scss

```diff
@@ -0,0 +1,27 @@
+.layout.main {
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
+
+  & > .container {
+    flex: 1;
+  }
+
+  & > footer {
+    min-height: 4.1rem;
+
+    .quick-links {
+      gap: 0 1.5em;
+
```

Comment:
Just my personal preference but I'm not a big fan of mixing `rem` and `em`, but I would love to hear if there's a reason to use them together. e.g. `.quick-links` gap is based on its parental element.

Reply:
Great question, and I don't have a strong reason in this particular case but I'll explain my idea:

If we change the base font size, then everything scales proportionally:

<img width="1895" alt="image" src="https://github.com/user-attachments/assets/52add04f-2170-40cc-92ec-c23c96262f4b">

If we just change the font size of the footer element, then the blue box stays the same height, but the font size changes:

<img width="1901" alt="image" src="https://github.com/user-attachments/assets/aa3f43d9-5de0-42ac-9273-e2818fa45ab9">

I figure the height of the blue box (in `rem`) is a layout element and we'd prefer if it stays roughly that same height, even if the font inside changes. But he spacing between the links (`em`) should scale with the font.

Here's an exaggerated version to demonstrate:

<img width="395" alt="image" src="https://github.com/user-attachments/assets/0039737c-d523-4752-8900-8df96eb2d763">

In this case it barely makes a difference, but sometimes it can have an impact. (If users zoom in the page like my mom does, or if they change their screen text scaling in their accessibility settings, etc). Mostly so we can control what shrinks and grows in an element if we change its font size.

Let me know if that makes sense! It doesn't do much in this case so I could change it if you'd prefer

Reply:
It does make sense! Thanks for your explanation 👏